slices/ca-certificates-java.yaml

```diff
@@ -28,6 +28,12 @@ slices:
       /etc/ssl/certs/java/:
       /usr/share/ca-certificates-java/ca-certificates-java.jar:
 
+  data:
+    essential:
+      - ca-certificates_data-with-certs
+  contents:
+    /etc/ssl/certs/java/:
```

Comment:
The indentation seems a bit odd there.

Reply:
I am quite surprised that the workflows did not pick up on this. They are treating it as another slice which they should, but I am surprised that they did not complain about the unknown field.
https://github.com/canonical/chisel-releases/actions/runs/10673178458/job/29581840492#step:8:53
```
INFO: Installing ca-certificates-java_contents on amd64...
```

Reply:
Pushed https://github.com/canonical/chisel-releases/pull/308/commits/8d5307ef4c4d5928adb47bbaf21e38acb02f5bba.